database/seeders/ShieldSeeder.php

```
@@ -145,9 +146,9 @@ protected static function makeRolesWithPermissions(string $rolesWithPermissions)
             return;
         }
 
-        /** @var \Illuminate\Database\Eloquent\Model $roleModel */
+        /** @var Model $roleModel */
         $roleModel = Utils::getRoleModel();
```

PHPStan (2.2.7)

[error] PHPDoc tag `@var` with type Illuminate\Database\Eloquent\Model is not subtype of native type string.


(varTag.nativeType)